offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/dataresources/persons/PersonDbConnector.groovy

```diff
@@ -783,6 +783,7 @@ class PersonDbConnector implements CreatePersonDataSource, SearchPersonDataSourc
 
   @Override
   Optional<Integer> findPerson(Person person) {
+    //Todo fix me: initialize as Optional and set Optional.empty() (see OfferDbConnector: getOffer(OfferId offerId) line 226)
```

Comment:
What is with this TODO?

Reply:
I wanted to suggest to initialize the variable personID as empty optional such as in the OfferDBConnector. But since it was not part of my PR I did not change it yet